packages/flutter_test/lib/src/window.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:isolate';
```

Comment:
Was this added on purpose?

Reply:
It automatically got added with the addition of the `sendPortPlatformMessage` method. The `SendPort` parameter is from `dart:isolate`.